app/models/file_search.rb

```diff
@@ -35,32 +38,32 @@ def search_term
     params[:q]
   end
 
-  def work_name
-    params[:work]
+  def page
+    params[:page]
   end
 
-  def venue_names
-    params.fetch(:venues) { all_venue_names }
+  def per_page
+    params[:per_page]
   end
 
-  def record_types
-    params.fetch(:record_types) { all_record_types }
+  def work_name
+    params[:work]
   end
 
-  def show_articles?
-    record_types.include?("articles")
+  def venue_names
+    params.fetch(:venues) { all_venue_names }
   end
 
-  def show_images?
-    record_types.include?("images")
+  def resource_types
+    resource_type.present? ? [resource_type.to_s] : []
   end
 
-  def show_audios?
-    record_types.include?("audios")
+  def types
```

Comment:
`types` vs `resource_types` is somewhat confusing naming.  Is there a better option?  I don't have a good one off the top of my head.